staging/src/k8s.io/kubectl/pkg/cmd/patch/patch.go

```diff
@@ -235,7 +240,9 @@ func (o *PatchOptions) RunPatch() error {
 		NamespaceParam(o.namespace).DefaultNamespace().
 		FilenameParam(o.enforceNamespace, &o.FilenameOptions).
 		Subresource(o.Subresource).
-		ResourceTypeOrNameArgs(false, o.args...).
+		ResourceTypeOrNameArgs(o.All, o.args...).
```

Comment:
field and label selectors in `patch` command sounds reasonable to me but I'm strongly reluctant to add `all` in here. Because this flag already creates confusions to users in other commands and I don't think, we'd want to add more. 

In addition to that all flag is also mutually exclusive with the other selectors https://github.com/kubernetes/kubernetes/blob/1cfb932111e53e296a3b0f001b6f5d03f212f32f/staging/src/k8s.io/cli-runtime/pkg/resource/builder.go#L585

Reply:
Agreed don't want to add all because it is highly confusing to users as it does not actually mean "all."

Reply:
Well, it means "select all" so you don't have to create dummy selector when you want to select all instances of the object. Would you prefer the flag name to be different? If so which one and how'd you see the consistency with other commands and reusing the builders?